boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -60,34 +59,159 @@ currently supports the following hardware features:
    * - GPIO
      - on-chip
      - gpio
+   * - ADC
+     - on-chip
+     - adc
+   * - I2C
+     - on-chip
+     - i2c
+   * - KSCAN
+     - on-chip
+     - kscan
+
+
+Other hardware features are not currently supported by Zephyr (at the moment)
+
+The default configuration can be found in the
+:zephyr_file:`boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig` Kconfig file.
+
+Programming and debugging on it83202
+************************************
+
+Wiring
+=======
+#. Connect your DownLoad Board to your host computer using the USB cable.
+
+   .. image:: ite_Downloadboard_setup.jpg
+        :width: 600px
+        :align: center
+        :alt: ITE Download Board Connected
 
-Programming and debugging
-*************************
+#. Connect your Download Board to J145 on the it8xxx2_evb board.
+
+   .. image:: it8xxx2_evb_setup.jpg
+        :width: 600px
+        :align: center
+        :alt: ITE Download Board Connected
+
+   .. note:: Be careful during connection!
+    Use separate wires to connect I2C pins with pins on the it8xxx2_evb board.
+    Wiring connection is described in the table below.
+
+    +-------------+---------------+
+    |   J5        | it8xxx2_evb   |
+    |   Connector | J145 Connector|
+    +=============+===============+
+    |      2      |       7       |
+    +-------------+---------------+
+    |      3      |       9       |
+    +-------------+---------------+
+    |      4      |       10      |
+    +-------------+---------------+
+
+#. Connect UART0 port of the it8xxx2_evb board
+   to your host computer using the usb cable.
+
+   .. note:: Be careful during connection!
+    Use separate wires to connect USB serial with pins on the it8xxx2_evb board.
+    Wiring connection is described in the table below.
+
+    +-------------+---------------+
+    |   USB       |  it8xxx2_evb  |
+    |   Connector |  Connector    |
+    +=============+===============+
+    |      TX     |   UART0 5     |
+    +-------------+---------------+
+    |      RX     |   UART0 7     |
+    +-------------+---------------+
+    |      GND    |      J15      |
+    +-------------+---------------+
+
+#. Apply power to the board via a USB cable.
 
 Building
 ========
 
-Applications for the ``it8xxx2_evb`` board configuration can be built as usual
-(see :`Zephyr Getting Started Guide`_):
+#. Build :ref:`hello_world` application as you would normally do
+   (see :`Zephyr Getting Started Guide`_):.
 
 .. zephyr-app-commands::
    :board: it8xxx2_evb
+   :zephyr-app: samples/hello_world
    :goals: build
 
+#. The file :file:`zephyr.bin` will be created if the build system
+   can find the image generation tool.
```

Comment:
Is the image generation tool separate from Zephyr?  Please provide instructions to get this tool.

Reply:
The image generation tool is west, modified.